tests/test_cxl_device_single_logical_device.py

```diff
@@ -31,15 +34,16 @@ async def test_single_logical_device_run_stop(get_gold_std_reg_vals):
     memory_size = 256 * MB
     memory_file = "mem.bin"
     transport_connection = CxlConnection()
-    device = CxlType3Device(
-        transport_connection,
+    device = SingleLogicalDevice(
         memory_size=memory_size,
         memory_file=memory_file,
-        dev_type=CXL_T3_DEV_TYPE.SLD,
+        test_mode=True,
+        cxl_connection=transport_connection,
     )
 
     # check register values after initialization
-    reg_vals = str(device.get_reg_vals())
+    # pylint: disable=protected-access
+    reg_vals = str(device._cxl_type3_device.get_reg_vals())
```

Comment:
Could you create `def get_reg_vals()` in `SingleLogicalDevice` which calls `_cxl_type3_device.get_reg_vals()`?

Reply:
This wouldn't get rid of `# pylint: disable=protected-access` due to the `for sld_device in device._sld_devices:` loop above.

If you also want me to wrap this with `def get_sld_devices()`, and get rid of that pylint entirely, let me know.

Reply:
Yes, let's do that. `get_sld_devices()`.